contrib/packaging/rpm/cfg/cilium.spec.envsubst

```diff
@@ -0,0 +1,70 @@
+%global commit0 ${COMMIT}
+%global shortcommit0 %(c=%{commit0}; echo ${c:0:7})
+
+Name:          cilium
+Version:       ${VERSION}
+Release:       0.git%{shortcommit0}%{?dist}
+Summary:       BPF & XDP for containers
+License:       Apache
+URL:           https://github/cilium/cilium
+ExclusiveArch: x86_64
+Source0:       https://github.com/cilium/%{name}/archive/%{commit0}.tar.gz#/%{name}-%{shortcommit0}.tar.gz
+BuildRequires: golang < 1.8.0, glibc-devel(x86-32)
+Requires:      docker-engine >= 1.12, clang >= 3.8, glibc-devel(x86-32), iproute >= 4.6, kernel >= 4.8
```

Comment:
Once package will land in repo it will stay there. Either installation based on direct links or `--allowerasing` will be problematic, I'll happy only when clang3.8 will be provided.